feasibility-portal/backend/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.7'
 services:
   feasibility-gui-backend:
-    image: ghcr.io/medizininformatik-initiative/feasibility-backend:2.0.0-rc1
+    image: ghcr.io/medizininformatik-initiative/feasibility-backend:2.1.0-RC2
```

Comment:
@DiCanio - will have to be changed again ones backend is updated to newest version

Reply:
I think we should do this in a separate issue. This only reflects necessary changes in order to be compatible with the new plugin version. In other words, this sets a minimum required version for the backend component.

Reply:
@DiCanio  sounds good